controllers/CalendarController.php

```diff
@@ -104,7 +104,7 @@ public function actionEdit($id = null)
         $model = ($id) ? $this->findModel($id) : new ExternalCalendar($this->contentContainer);
 
         try {
-            if ($model->load(Yii::$app->request->post())) {
+            if ($model->load(Yii::$app->request->post()) && $model->validate()) {
                 (new ICalSync(['calendarModel' => $model, 'skipEvents' => true]))->syncICal();
                 if(!$model->hasErrors()) {
```

Comment:
Shouldn't this already be covered by this line?

Reply:
AFAIK
load only pupolates the attributes, no validation is performed on it, you can try to test it out, in order to see difference.
it contines the execution only after all the $_POST datat is valid.
`$model->load(\Yii::$app->request->post()); // $model->attributes = \Yii::$app->request->post('ContactForm');`
[Validating Input](https://www.yiiframework.com/doc/guide/2.0/en/input-validation)

Reply:
You are right, I missread the line.